go.mod

```diff
@@ -2,6 +2,12 @@ module github.com/liftbridge-io/liftbridge
 
 go 1.13
 
+replace github.com/liftbridge-io/liftbridge-api => ../liftbridge-api
```

Comment:
these should replace with github.com/ably-forks/liftbridge

Reply:
Yes, I'll have to merge the other PR in ably-forks/liftbridge-api before that though.